mod.py

```diff
@@ -195,8 +195,7 @@ def __floordiv__(self, other):
                 'integer division by {}'.format(converted)
             )
 
-        inverted = (other * self.inverse)
-        return inverted.inverse
+        return self._value* converted.inverse
```

Comment:
just to make the style-checker happy, could you add a space before the `*` ? :-)